Client/Frontend/BraveVPN/BraveVPN.swift

```diff
@@ -645,4 +648,48 @@ class BraveVPN {
       }
     }
   }
+  
+  static var shouldProcessVPNAlerts: Bool {
+    if !Preferences.PrivacyReports.captureVPNAlerts.value {
+      return false
+    }
+    
+    switch vpnState {
+    case .installed(let enabled):
+      return enabled
+    default:
+      return false
+    }
+  }
+
+  static func processVPNAlerts() {
+    if !shouldProcessVPNAlerts { return }
+
+    Task {
+      let (data, success, error) = await GRDGatewayAPI.shared().events()
+      if !success {
+        log.error("VPN getEvents call failed")
+        if let error = error {
+          log.warning(error)
+        }
+
+        return
+      }
+
+      guard let alertsData = data["alerts"] else {
+        log.error("Failed to unwrap json for vpn alerts")
+        return
+      }
+
+      do {
+        let dataAsJSON =
+          try JSONSerialization.data(withJSONObject: alertsData, options: [.fragmentsAllowed])
```

Comment:
`.fragmentsAllowed` not needed :) The `BraveVPNAlertJSONModel` is a dictionary, so no need here.